io.github.junalmeida.webapps_manager.json

```diff
@@ -0,0 +1,55 @@
+{
+    "id": "io.github.junalmeida.webapps_manager",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.PySide.BaseApp",
+    "base-version": "6.7",
+    "command": "webapps_manager",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=host-os",
+        "--filesystem=/var/lib/flatpak",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Flatpak"
+    ],
+    "modules": [
+        "python3-modules.json",
+        {
+            "name": "debugpy",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
```

Comment:
this is not allowed. It will NOT work.

Reply:
What exactly will not work, `debugpy` or `python3-modules.json`? I tested it locally and both work. I'm able to run the app with the dependencies listed in the json file, as well as debug it.